src/components/Modal/Modal.vue

```diff
@@ -703,51 +710,63 @@ $header-size: 50px;
 	/* Content */
 	.modal-container {
 		display: block;
-		overflow: hidden;
+		overflow: auto; // avoids unecessary hacks if the content should be bigger than the modal
 		padding: 0;
 		transition: transform 300ms ease;
 		border-radius: var(--border-radius-large);
 		background-color: var(--color-main-background);
 		box-shadow: 0 0 40px rgba(0, 0, 0, .2);
 	}
-	&:not(&--large):not(&--full) .modal-container {
-		max-width: 900px;
-		max-height: 80%;
-	}
 
 	// Sizing
-	&--full {
+	&--small {
 		.modal-container {
-			max-width: 100%;
-			max-height: 100%;
-			border-radius: 0;
+			width: 350px;
+			height: 200px;
 		}
 	}
-	&--full,
-	&--spread-navigation {
-		.prev,
-		.next {
-			position: absolute;
-			width: 8%;
-			height: 35vw;
-		}
-		.prev {
-			left: 0;
+	&--medium {
+		.modal-container {
+			width: 500px;
+			height: 600px;
 		}
-		.next {
-			right: 0;
+	}
+	&--normal {
+		.modal-container {
+			max-width: 70%;
+			width: 700px;
+			max-height: 80%;
+			height: 600px;
 		}
 	}
 	&--large {
 		.modal-container {
-			max-width: 85%;
+			max-width: 90%;
+			width: 900px;
 			max-height: 90%;
+			height: 700px;
+		}
+	}
+	&--full {
+		.modal-container {
+			width: 100%;
+			height: calc(100% - $header-size);
+			position: absolute;
+			top: $header-size;
```

Comment:
We could have gone for 
```suggestion
			height: auto;
			position: absolute;
			top: $header-size;
			bottom: 0;
```

Reply:
Which imho is better than calc with `$header-size` :)
@szaimen

Reply:
Would the result be the same?